src/ui/symbols.rs

```diff
@@ -38,7 +38,8 @@ impl Default for Symbols {
                         j/k/h/l OR Up/Down/Left/Right: Scroll
 
                         ## Global
-                        R: Start clash service
+                        R: Restart clash service
+                        r: Restart clash core
```

Comment:
Using uppercase for global hotkeys would make it more consistent. How about `E` (rEstart).

Reply:
I think just sending a restart request might be more often, given that restart a system service need to type in password?
And it might not work if the clash service is started as a user one.
Given these, name R to "Restart clash core", E to "rEstart clash service" might be better?

Reply:
I actually use 'restart' instead of 'start'. The function i want to make is 'Start', so the `help message` is `Start ClashSrv`.

Reply:
> I think just sending a restart request might be more often, given that restart a system service need to type in password? And it might not work if the clash service is started as a user one. Given these, name R to "Restart clash core", E to "rEstart clash service" might be better?

Ok, `Stop` and `Start` is not often.

Reply:
> I think just sending a restart request might be more often, given that restart a system service need to type in password? And it might not work if the clash service is started as a user one. Given these, name R to "Restart clash core", E to "rEstart clash service" might be better?

I use mihomo lightly. I don't understand why need to restart the mihomo service? Can you show me some use cases?

Reply:
> > I think just sending a restart request might be more often, given that restart a system service need to type in password? And it might not work if the clash service is started as a user one. Given these, name R to "Restart clash core", E to "rEstart clash service" might be better?
> 
> I use mihomo lightly. I don't understand why need to restart the mihomo service? Can you show me some use cases?

you are using systemctl restart, just see codes:
```rust
let output = Command::new("systemctl")
                    .arg("restart")
                    .arg(self.clash_srv_name.as_str())
                    .output()?;

                return Self::string_process_output(output);
```
this means restart the clash service

Reply:
if you want to set start the service ,just use `systemctl start mihomo`

Reply:
I know the `start`. If use `restart`, i have `start` and `restart` function. Sometime i need `restart` to test something.